packages/fast-components-react-base/src/utilities/extractElement.ts

```diff
@@ -0,0 +1,26 @@
+import ReactDOM from "react-dom";
+import { isNil } from "lodash-es";
+
+/**
+ * Attempts to extract an html element from a ref
+ */
+export function extractElementFromRef(
+    sourceRef: React.RefObject<any> | HTMLElement
```

Comment:
I think this should be `React.RefObject<HTMLElement>`

Reply:
yep.  Moved to its own pr
https://github.com/microsoft/fast-dna/pull/2613